jf_agent/git/gitlab_adapter.py

```diff
@@ -402,7 +406,7 @@ def _normalize_short_form_repo(api_repo, redact_names_and_urls):
 
 
 def _normalize_commit(
-    api_commit, normalized_repo, strip_text_content: bool, redact_names_and_urls: bool
+    api_commit, normalized_repo, strip_text_content: bool, redact_names_and_urls: bool, api_vrsn,
```

Comment:
Looks like you never ended up using this new arg; can yank it

Reply:
done (& tested)